generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,533 @@
+But if you're curious, one of the most common methods is called [maximum likelihood estimation](https://en.wikipedia.org/wiki/Maximum_likelihood_estimation).
+
+</div>
+
+If we estimate the model with the logit link function, and then convert the predictions back to a 0-1 scale, we get something like this:
+
+![The same scatterplot showing temperature on the x-axis and sepsis (0 or 1) on the y-axis, but the line of best fit is now a sigmoidal curve, approaching but never reaching 0 for low values of temperature, and then curving up gently as temperature increases.](media/logit_prediction.png)
+
+``` -R code for the above plot
+ggplot(data, aes(y=sepsis, x=temp)) + 
+  geom_point() + 
+  theme_bw() + 
+  labs(y = "Sepsis", x = "Temperature") + 
+  scale_y_continuous(breaks = c(0,1))  + 
+  stat_smooth(method = "glm", method.args = list(family = "binomial"))
+```
+
+That looks much better!
+The predicted values now respect the boundaries on our outcome variable -- they can never go below 0 or above 1. 
+
+## Other Link Functions
+
+Logistic regression (regression using a logit link function) works well for binary outcomes.
+But what about other kinds of outcome variables?
+
+If your outcome is counts of something, you can use a **Poisson** or **Negative Binomial** link function. 
+For example, perhaps your outcome is the number of [Adverse Childhood Experiences (ACEs)](https://www.cdc.gov/violenceprevention/aces/fastfact.html) per patient. 
+It's not possible for someone to have fewer than 0 ACEs, so that variable is bound at 0. 
+It's also measured as a count (how many total ACEs experienced), so no one will have 1.2 ACEs, for example. 
+
+For outcomes bounded at 0 and 1 (like a probability or proportion), you can use logistic regression, but a **beta** link function may be more appropriate.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For more details on beta regression and when to use it, see [Analysing continuous proportions in ecology and evolution: A practical introduction to beta and Dirichlet regression](https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.13234) by Douma and Weedon (2019).
+
+</div>
+
+If your outcome is positively skewed, especially something bound at 0, then you can use a **Gamma** or **Inverse-Gaussian** link function. 
+These are especially useful for outcomes like cost, income, or length of stay -- continuous variables bound at 0 and usually positively skewed.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+To learn more about how to talk about the shape of a variable, such as identifying positive and negative skew, see the [Khan Academy lesson on shapes of distributions](https://www.khanacademy.org/kmap/measurement-and-data-j/md231-data-distributions/md231-displays-of-distributions/v/shapes-of-distributions).
```

Comment:
Not a review suggestion, I just thought you might find this delightful: 

my research stats prof in undergrad taught us to remember how to talk about skew by thinking about the fact that it's named after the outliers that "ruined" the normal distribution. Like ["I would have gotten away with being normally distributed too, if it weren't for you meddling outliers"](https://scoobydoo.fandom.com/wiki/List_of_%22And_I_Would_Have_Gotten_Away_With_It_Too,_If_It_Weren%27t_For_You_Meddling_Kids%22_Quotes) 

Of course, I promptly filed this mnemonic device away in my head under the name "Skew-by Doo".